trace-shield/helm/trace-shield/templates/oathkeeper-rules.yaml

```diff
@@ -7,50 +7,199 @@ metadata:
 data:
   access-rules.yaml: |-
     {{- if  .Values.config.mimir.enabled }}
```

Comment:
Yeah this file can be cleaned up a lot, but since I wasn't sure yet if it would work I kept it stupid simple for now. We can create a template function that will make configuring this much simpler.